spring-integration-mail/src/main/resources/org/springframework/integration/mail/config/spring-integration-mail.xsd

```diff
@@ -167,10 +167,11 @@
 					<xsd:attribute name="task-executor" type="xsd:string">
 						<xsd:annotation>
 							<xsd:documentation><![CDATA[
-								Reference to a bean that implements
+								[DEPRECATED] Reference to a bean that implements
```

Comment:
Same comment re: deprecation; I don't see how we can deprecate it, but just ignore it.

Reply:
I saw that; but it makes no sense to me; if the parser ignores it, we should remove it altogether (and record it as a breaking change in the migration guide).

Reply:
OK. Let me issue an new PR against `6.0.x` to just deprecate this executor.
Then I'll fix this as a removal.
And yes: in the end a dedicated migration guide note.